avocado/utils/software_manager/backends/rpm.py

```diff
@@ -6,7 +6,8 @@
 from ... import process
 from .base import BaseBackend
 
-log = logging.getLogger('avocado.utils.software_manager')
+
+log = logging.getLogger('avocado.test')
```

Comment:
For consistency.: s/log/LOG/

Reply:
Probably this is a cache issue, it was already fixed.